src/accommodation/accommodation.controller.ts

```diff
@@ -113,7 +113,7 @@ export class AccommodationController {
     },
   })
   @ApiBearerAuth()
-  @Post('/file/:id')
```

Comment:
could you rename the endpoint. 'file' is to generic name. better rename it to preview-upload or smth like that. There is  possible other files.